src/xdocs/cmdline.xml.vm

```diff
@@ -96,26 +96,27 @@ java -D&lt;property&gt;=&lt;value&gt;  \
           <code>-s</code> option. Default value is 8.
         </li>
         <li>
-          <code>-t, --tree</code> - Prints Abstract Syntax Tree(AST) of the checked file. The option
-          cannot be used other options and requires exactly one file to run on to be specified.
+          <code>-t, --tree</code> - This option is used to display the Abstract Syntax Tree (AST)
+          of the specified file. It can only be used on a single file and cannot be combined
+          with other options.
         </li>
         <li>
-          <code>-T, --treeWithComments</code> - Prints Abstract Syntax Tree(AST) with comment nodes
-          of the checked file. The option cannot be used with other options and requires exactly
-          one file to run on to be specified.
+          <code>-T, --treeWithComments</code> - This option is used to display the Abstract Syntax
+          Tree (AST) <b>with comment nodes</b> of the specified file. It can only be used on a
+          single file and cannot be combined with other options.
         </li>
         <li>
-          <code>-J, --treeWithJavadoc</code> - Prints Abstract Syntax Tree(AST) with Javadoc nodes
-          and comment nodes of the checked file. Attention that line number and columns will not be
-          the same as it is a file due to the fact that each javadoc comment is parsed separately
-          from java file. The option cannot be used with other options and requires exactly one
-          file to run on to be specified.
+          <code>-J, --treeWithJavadoc</code> - This option is used to display the Abstract Syntax
+          Tree (AST) with Javadoc nodes of the specified file. It can only be used on a single
+          file and cannot be combined with other options. Note that the line number and columns
+          will not be the same due to the fact that each javadoc comment is
```

Comment:
`Note that the line number and columns will not be the same due to` => `Note that the line and column numbers between the 2 will not be in sync with the file's numbering due to`

It wasn't clear what would not be the same in your original statement. It originally sounded like the line numbers would be not be the same as the column numbers.

Also show a small quick example to confirm that this is the case.

Reply:
@rnveach , I should've tested the options before..

```
E:\MANISH\Workspace>cat Test.java
/**
 * Some javadoc.
 *
 * @author MANISH-K-07
 * @version 1.0
 */
class Test {
        /*
          romani
          rnveach
          nrmancuso
        */
        // MANISH-K-07
}

E:\MANISH\Workspace>java -jar checkstyle-10.13.0-all.jar -J Test.java
COMPILATION_UNIT -> COMPILATION_UNIT [7:0]
`--CLASS_DEF -> CLASS_DEF [7:0]
    |--MODIFIERS -> MODIFIERS [7:0]
    |--BLOCK_COMMENT_BEGIN -> /* [1:0]
    |   |--COMMENT_CONTENT -> *\n * Some javadoc.\n *\n * @author MANISH-K-07\n * @version 1.0\n  [1:2]
    |   |   `--JAVADOC -> JAVADOC [1:3]
    |   |       |--NEWLINE -> \n [1:3]
    |   |       |--LEADING_ASTERISK ->  * [2:0]
    |   |       |--TEXT ->  Some javadoc. [2:2]
    |   |       |--NEWLINE -> \n [2:16]
    |   |       |--LEADING_ASTERISK ->  * [3:0]
    |   |       |--NEWLINE -> \n [3:2]
    |   |       |--LEADING_ASTERISK ->  * [4:0]
    |   |       |--WS ->   [4:2]
    |   |       |--JAVADOC_TAG -> JAVADOC_TAG [4:3]
    |   |       |   |--AUTHOR_LITERAL -> @author [4:3]
    |   |       |   |--WS ->   [4:10]
    |   |       |   `--DESCRIPTION -> DESCRIPTION [4:11]
    |   |       |       |--TEXT -> MANISH-K-07 [4:11]
    |   |       |       `--NEWLINE -> \n [4:22]
    |   |       |--LEADING_ASTERISK ->  * [5:0]
    |   |       |--WS ->   [5:2]
    |   |       |--JAVADOC_TAG -> JAVADOC_TAG [5:3]
    |   |       |   |--VERSION_LITERAL -> @version [5:3]
    |   |       |   |--WS ->   [5:11]
    |   |       |   `--DESCRIPTION -> DESCRIPTION [5:12]
    |   |       |       |--TEXT -> 1.0 [5:12]
    |   |       |       |--NEWLINE -> \n [5:15]
    |   |       |       `--TEXT ->   [6:0]
    |   |       `--EOF -> <EOF> [6:1]
    |   `--BLOCK_COMMENT_END -> */ [6:1]
    |--LITERAL_CLASS -> class [7:0]
    |--IDENT -> Test [7:6]
    `--OBJBLOCK -> OBJBLOCK [7:11]
        |--LCURLY -> { [7:11]
        |--BLOCK_COMMENT_BEGIN -> /* [8:1]
        |   |--COMMENT_CONTENT -> \n\t  romani\n\t  rnveach\n\t  nrmancuso\n\t [8:3]
        |   `--BLOCK_COMMENT_END -> */ [12:1]
        |--SINGLE_LINE_COMMENT -> // [13:1]
        |   `--COMMENT_CONTENT ->  MANISH-K-07\n [13:3]
        `--RCURLY -> } [14:0]

E:\MANISH\Workspace>
```
From what I can see,  the line nos are in sync.. just the column numbers are differing by +2 as tree is parsed considering leading asterix position as 0

Reply:
> From what I can see, the line nos are in sync.. just the column numbers are differing by +2 as tree is parsed considering leading asterix position as 0

I don't know how others feel, but I would report this as an issue. It seems to me if we go out of the way to sync line numbers, we should do the same for columns.

Also please point out for those who are not seeing it where the de-sync is and what it should be.

Let me know when documentation in this PR is updated.

Reply:
@rnveach , please see Issue #14780 
I've detailed everything in the issue description. 

For now, I've updated the docs to mention just the differing column numbers.